test/src/main/java/org/apache/accumulo/test/ShellIT.java

```diff
@@ -352,10 +352,11 @@ public void authsTest() throws Exception {
   }
 
   @Test
-  public void userTest() {
+  public void userTest() throws IOException {
     Shell.log.debug("Starting user test --------------------------");
-    // Test cannot be done via junit because createuser only prompts for password
-    // exec("createuser root", false, "user exists");
+    exec("createuser root", false, "user exists");
+    exec("createuser test", true);
+    exec("createuser test", false, "user exists");
```

Comment:
Should probably use a unique user name for this, to reduce the liklihood of collision with another test. The base class should be able to provide a unique name, based on the name of the test method.

Reply:
Could also delete the created user when done.

Reply:
Good ideas. I will implement.